### ISSUE TITLE
sap_general_preconfigure: Incorrect variable inclusion after commit e97e3fb0

### ISSUE THREAD

**Issue by berndfinger** [sap-linuxlab/community.sap_install#146]
The minor release specific variable inclusion does not work correctly for RHEL after the following line:
```
- '{{ ansible_distribution.split("_")[0] }}_{{ ansible_distribution_major_version }}.yml'
```
was added to file tasks/main.yml. On RHEL 8.0 or 8.1, this code will first find file vars/RedHat_8.yml instead of the desired RedHat_8.1.yml or RedHat_8.0.yml.

**Comment by berndfinger** [comment]
If there is a case where `ansible_distribution` contains an underscore and we need to take the part of the string to the left of the underscore, the problem can be solved by adding the following line after `with_first_found:`:
```
- '{{ ansible_distribution.split("_")[0] }}_{{ ansible_distribution_version }}.yml'
```
and removing the two similarly looking lines of the `with_first_found` condition which do not contain the split function.

**Comment by berndfinger** [comment]
The content of `ansible_distribution` for "SLES for SAP Applications" is `SLES_SAP`.

**Comment by berndfinger** [comment]
Fixed in [d93938a8163abf00feb8f5747fdafe9a2e4d2d97](https://github.com/berndfinger/community.sap_install/commit/d93938a8163abf00feb8f5747fdafe9a2e4d2d97)

**Comment by berndfinger** [comment]
Note: Splitting a string which does not have an underscore results in the complete string to be taken, so `ansible_distribution.split("_")[0]` will return `RedHat` in the case of a RHEL system.
